shared/charts/backup-exporter/values.yaml

```diff
@@ -4,7 +4,7 @@
 
 image:
   repository: google/cloud-sdk
-  tag: 241.0.0-alpine
+  tag: 242.0.0-alpine
```

Comment:
I can't find the overrides to use the values from versions.yml  instead of these defaults.

Reply:
Fixed in https://github.com/gpii-ops/gpii-infra/pull/400/commits/f808e1011486411176e9500a52f3455aa8b41441